pkg/nfd-master/nfd-master.go

```diff
@@ -30,18 +30,19 @@ import (
 	"strings"
 	"time"
 
+	"k8s.io/client-go/rest"
+
 	"github.com/davecgh/go-spew/spew"
 
-	v1alpha1 "github.com/swatisehgal/topologyapi/pkg/apis/topology/v1alpha1"
-	clientset "github.com/swatisehgal/topologyapi/pkg/generated/clientset/versioned"
+	"github.com/swatisehgal/topologyapi/pkg/apis/topology/v1alpha1"
+	topologyclientset "github.com/swatisehgal/topologyapi/pkg/generated/clientset/versioned"
 	"golang.org/x/net/context"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/credentials"
 	"google.golang.org/grpc/peer"
 	api "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/client-go/rest"
```

Comment:
"k8s.io/client-go/rest" removed but used later

Reply:
yep it moved above, I updated imports again